lib/fluent/env.rb

```diff
@@ -25,6 +27,7 @@ module Fluent
   DEFAULT_OJ_OPTIONS = Fluent::OjOptions.load_env
   DEFAULT_DIR_PERMISSION = 0755
   DEFAULT_FILE_PERMISSION = 0644
+  COMMON_UUID = ENV['FLUENT_COMMON_UUID'] || SecureRandom.uuid
```

Comment:
I don't know what it's for. 
I think `COMMON_UUID` should be more specific.

Reply:
This is a unique ID for that Fluentd instance (the supervisor and workers).

What names can we think of?

Reply:
At least, I think  FLUENT_COMMON_UUID is not appropriate...

Reply:
Hmm, wouldn't it be natural to have a common ID as an instance?

Currently, this is only used for the `BUFFER_DIR_NAME` of `SourceOnlyBufferAgent`.

https://github.com/fluent/fluentd/blob/0e96d5485912a099c0e445b62bbab9963c579601/lib/fluent/source_only_buffer_agent.rb#L21-L24

However, I feel there would be no need to limit its use.

If we limit this ID to this buffer name, it would be better not to define it in `env.rb`.
I think it would be also somewhat inappropriate to put such a limited thing into the entire environment variables.

Then, there would be no good way to share the same temporal directory among workers.
It would be because workers did not have a common unique ID between them.

I don't know if it will actually be used for other purposes in the future, but it wouldn't be so bad to have a unique ID for the instance as a whole.

Reply:
@kenhys gave us advice.
`FLUENT_INSTANCE_ID` and `Fluent::INSTANCE_ID` would be better.

Reply:
Fixed.